.github/wiki/Standard-File-Organization.md

```diff
@@ -146,6 +164,10 @@ This folder contains the main sources of the project. There are three sub-folder
 
 This folder contains the unit testing and integration testing of the main project.
 
+## {ProjectName}KIFUITests
+
+This folder contains the KIF UI testing of the main project. Use KIF instead of UITests for speed and reliability.
```

Comment:
```suggestion
This folder contains the KIF UI testing of the main project. Use KIFUITests instead of UITests for speed and reliability.
```

Reply:
@Shayokh144 
The sentence is more on the naming of the tools (KIF and XCUITest). Would it be clearer like this
```
This folder contains the KIF UI testing of the main project. Use KIF instead of XCUITest for speed and reliability.
```

Reply:
this will be more readable I think.